src/components/pages/course-page/components/lesson/Lesson.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, useMemo } from 'react';
+import { Lesson as LessonInterface } from '@/types/common/course';
+import styles from './Lesson.module.scss';
+import {
+  Collapse,
+  Divider,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import { ExpandMore, Lock } from '@mui/icons-material';
+import VideoPlayer from '@/components/common/video-player/VideoPlayer';
+import { parseLessonPoster, parseTime } from '@/utils';
+
```

Comment:
Space should be removed in order to previous styles in components

Reply:
fixed